torch/_torch_docs.py

```diff
@@ -5380,6 +5380,22 @@ def merge_dicts(*dicts):
     ),
 )
 
+add_docstr(
```

Comment:
nit: we might want to add the same for e and pi given https://github.com/pytorch/pytorch/blob/0246b28510f11364d0e1fb7cc4eb353bd649a8f5/torch/__init__.py#L1681